content/en/docs/languages/js/instrumentation.md

```diff
@@ -1275,9 +1290,8 @@ const resource = Resource.default().merge(
 
 const metricReader = new PeriodicExportingMetricReader({
   exporter: new ConsoleMetricExporter(),
-
-  // Default is 60000ms (60 seconds). Set to 3 seconds for demonstrative purposes only.
-  exportIntervalMillis: 3000,
+  // Default is 60000ms (60 seconds). Set to 5 seconds for demonstrative purposes only.
+  exportIntervalMillis: 5000,
```

Comment:
A general comment since it has been coming up lately: to me it's a bad smell that we're repeating the default value in several places in the docs. This will be a pain to update if the default happens to change. How about something like:

```suggestion
  exportIntervalMillis: 5000, // Reduce export interval for demo purposes only.
```

I'd rather that we link to a page that says what the default is instead of repeating the default value.

Reply:
Also, I think that we agreed elsewhere that this would be ~ 10 sec?

Reply:
> in the PR for java who discusses to introduce that

Also, I think that we agreed elsewhere that this would be ~ 10 sec?

Reply:
let's have a dedicated issue to make sure that we have some consistency here.